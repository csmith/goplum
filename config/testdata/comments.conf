# comment
defaults { # comment
    alerts = ["text", "foo", "bar", ] #comment
    # comment
	good_threshold = 3
	interval = 1m30s


}

# comment
alert twilio.message "text" {
    sid = "xxx"
    token = "yyy"
    from = "+44 ..."
    to = "+44 ...."
}

# check http.get "chameth.com" {
#     url = "https://chameth.com/"
#     content = "Personal Homepage of Chris Smith"
# }
#
