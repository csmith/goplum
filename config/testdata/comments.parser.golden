{
  "DefaultSettings": {
    "alerts": [
      "text",
      "foo",
      "bar"
    ],
    "good_threshold": 3,
    "interval": 90000000000
  },
  "AlertBlocks": [
    {
      "Name": "text",
      "Type": "twilio.message",
      "Settings": {
        "from": "+44 ...",
        "sid": "xxx",
        "to": "+44 ....",
        "token": "yyy"
      }
    }
  ],
  "CheckBlocks": null,
  "PluginSettings": null
}